unix-alpha/CONTRIBUTING.md

```diff
@@ -41,28 +44,12 @@ We use the following tools:
 * `isort` to sort imports (`pipenv run isort .`)
 * `flake8` or `pylint` for code linting (`pipenv run flake8
   src/machine_stats/*` or `pipenv run pylint src`)
-* `bump2version` for version bumps (`pipenv run bump2version`)
 
 ### How to bump Machine Stats version
 
-We use [`bump2version`](https://pypi.org/project/bump2version/) to update version numbers.
-
-For example, if the current version in `1.0.0`:
-
-* `pipenv run bump2version patch` will change the version to `1.0.1`
-* `pipenv run bump2version minor` will change the version to `1.1.0`
-* `pipenv run bump2version major` will change the version to `2.0.0`
-
-After that all you need to do is to run
-
-```console
-git push origin master
-git push origin --tags
-```
-
-This will update the version and trigger the PyPI build and deploy.
-
-_Note_: You can verify that the version has been updated after running the `bump2version` command by checking the `config.cfg` file. (current_version)
+We use tags to release a new version of machine stats alpha.
+To make a new release, simply create a tag on the `alpha` branch and the
```

Comment:
@pranavmalaviya what do you think if we add a bit more information about how to create the release/tag. Something along this lines https://github.com/tidalmigrations/db-analyzer#instructions
Check step 4.